census/census_test.go

```diff
@@ -2,17 +2,22 @@ package census
 
 import (
 	"bytes"
+	"context"
+	"encoding/gob"
 	"encoding/json"
 	"math/big"
 	"testing"
+	"time"
 
 	"github.com/ethereum/go-ethereum/common"
+	qt "github.com/frankban/quicktest"
 	"go.vocdoni.io/dvote/api/censusdb"
 	"go.vocdoni.io/dvote/censustree"
 	"go.vocdoni.io/dvote/data/compressor"
+	"go.vocdoni.io/proto/build/go/models"
 )
 
-var internalAddresses = map[common.Address]*big.Int{
+var MonkeysAddresses = map[common.Address]*big.Int{
```

Comment:
what is `MonkeysAddresses` ?

Reply:
the holder addresses of our test token called "Monkeys"

Reply:
Let the naming battle begin :P

Reply:
TokenHolders is already the name of a struct :(